jest.config.js

```diff
@@ -2,13 +2,59 @@
 // https://jestjs.io/docs/en/configuration.html
 
 module.exports = {
+  preset: "ts-jest/presets/default",
+  moduleFileExtensions: [
+    "js",
+    "tsx",
+    "ts",
+    "json"
+  ],
+  globals: {
+    "ts-jest": {
+      "tsConfig": "tsconfig-jest.json",
+      "isolatedModules": true
+    }
+  },
+  transform: {
+    ".(jsx|js|ts|tsx)": "ts-jest"
+  },
   clearMocks: true,
-  roots: [
-    "build/commands/lib",
-    "build/commands/scripts"
+  resetMocks: true,
+  resetModules: true,
+  collectCoverage: true,
+  collectCoverageFrom: [
+    "<rootDir>/build/commands/lib/*",
+    "<rootDir>/components/**/**/*.ts",
+    "<rootDir>/components/**/**/*.tsx",
+    "!<rootDir>/components/definitions/*",
+    "!<rootDir>/components/**/constants/*",
+    "!<rootDir>/components/**/reducers/index.ts",
+    "!<rootDir>/components/**/store.ts",
+    "!<rootDir>/components/test/*",
+    "!<rootDir>/build/commands/lib/start.js",
```

Comment:
This one is being ignored as including it produces: 

```
STACK: SyntaxError: /Users/ryanlanese/Projects/brave-browser/src/brave/build/commands/lib/start.js: Legacy octal literals are not allowed in strict mode (123:37)

  121 |         else if (process.platform === 'darwin') {
  122 |             outputPath = outputPath + '_helper';
> 123 |             fs.chmodSync(outputPath, 0755);
      |                                      ^
  124 |         }
  125 |     }
```

I didn't want to attempt to modify that file as part of this pull request, and I think this is one is fine to ignore, as it doesn't cause any issues that I know of